DontForget/Sources/Presentations/Application/Creation/CreationUIView.swift

```diff
@@ -220,23 +258,25 @@ struct MemoView: View {
                 .padding(.leading, 16)
                 .padding(.bottom, 32)
                 .foregroundColor(.white)
-
-            TextField("간단한 메모를 하세요.(최대 30자)",
-                      text: $memo)
+            
+            TextField("", text: $memo,
+                      prompt: Text("가족여행 미리 계획하기").foregroundColor(.gray700))
```

Comment:
아하! 인자가 여러 개일 때 개행은 괄호를 열자마자 하기로 했어요 닫는 괄호 라인을 맞춰주기 위해!~!
```suggestion
            TextField(
                "",
                text: $memo,
                prompt: Text("가족여행 미리 계획하기").foregroundColor(.gray700)
            )
```

Reply:
넵! 죄송함다!ㅎ,ㅎ